pkg/services/pluginsettings/ifaces.go

```diff
@@ -7,8 +7,15 @@ import (
 )
 
 type Service interface {
+	// GetPluginSettings returns all Plugin Settings for the provided Org
```

Comment:
Nit. Maybe name this file `pluginsettings.go`. Not sure that's a decided convention we have, but feels like we use this in multiple places. Probably easier to find when searching for pluginsettings.